internal/pkg/model/template.go

```diff
@@ -20,6 +20,12 @@ type TemplateRepository struct {
 	Url        string                 `json:"url,omitempty" validate:"required_if=Type git"`
 	Ref        string                 `json:"ref,omitempty" validate:"required_if=Type git"`
 	WorkingDir string                 `json:"-"` // only for RepositoryTypeWorkingDir
+	Author     TemplateAuthor         `json:"-"`
```

Comment:
V CLI, v project manifest to nepotrebujeme, preto `json:-`.
Pridal som to sem, aby som z `DefaultRepository` mohol jednoducho vratit autora.